docs/data_api.md

```diff
@@ -12,13 +12,20 @@ Before continuing with the next section, it would be a good idea to have a look
 
 Processes running as part of your application architecture that update context information using REST operations that the Context Broker GE exports, are said to play a **Context Producer** role.  As an example, let’s consider an application for rating restaurants (let’s call it NiceEating). The client part of that application running on the smartphone of users would play the role of Context Producer, enabling them to rate restaurants.
 
+From a multitenancy point of view, let's consider that the service and subservice corresponding to the application are
+"smartown" and "NiceEating" respectively. All the requests shown in the examples below will include the `Fiware-service` 
+and `Fiware-servicepath` headers corresponding to that. From the point of view of the platform, these headers are
+mandatory. More information on multitency in [this specific section in the documentation](multitenancy.md).
+
 On the other hand, processes running as part of your application architecture that query context information using REST operations that the Context Broker GE exports are said to play a **Context Consumer** role.  Continuing with our NiceEating app, the mobile application running on the smartphone of users and enabling them to query for the rating of restaurants would play the role of Context Consumer.  Note that a given part of your application may play both the Context Producer and Context Consumer roles.  This would be the case of the mobile client of the NiceEating app enabling end users to rate, and query about rates of, restaurants.
 
-Entities that would be relevant to the NiceEating application are of type Restaurant, Client and Rating. For example, when a given user scores a restaurant (e.g. in a scale from 0 to 5, “Client1234” scores “4” for the “LeBistro” restaurant) the smartphone application plays the Context Producer role **creating** a Rating entity by issuing the following HTTP request :
+Entities that would be relevant to the NiceEating application are of type Restaurant, Client and Rating. For example, when a given user scores a restaurant (e.g. in a scale from 0 to 5, “Client1234” scores “4” for the “LeBistro” restaurant) the smartphone application plays the Context Producer role **creating** a Rating entity by issuing the following HTTP request:
 
 ```
 POST <cb_host>:<cb_port>/v2/entities
 Content-Type: application/json
+Fiware-service: smartown
```

Comment:
I'm unsing the same sample service that in other places in the doc (e.g. http://fiware-iot-stack.readthedocs.io/en/release-v4.1/multitenancy/index.html) in order to keep documentation consistent. I recommend other contributors do the same.

NTC (just informative)